### ISSUE TITLE
Screen reader states the form input field is required twice

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#2527]
## Builds

- [staging](https://console.cloud.google.com/cloud-build/builds;region=global/474aceaf-dcfb-4c5e-93ed-a4946f4f5d8d?project=proudcity-1184)
- [prod](https://console.cloud.google.com/cloud-build/builds;region=global/de1d036c-3dbe-4d7a-9166-c189cc74958e?project=proudcity-1184)

[Dev notes](https://github.com/proudcity/developers/blob/main/Github%20Issue%20Notes/2527%20-%20Required%20Form%20twice%20for%20screen%20readers.md)

## Source

https://airtable.com/appRFUc45sF4yWFut/tbl51inE74gEWlB0x/viwG9G4vAjsjQEh4g/recccSglFvqcXfa1k?blocks=hide

## Describe the bug

The screen reader states the form input field is required twice.  This is because the text "required" is visible on the form and the form input field has aria-required="true".

## Recommendation

If the field's label already contains the word "required",  you don't need aria-required attribute. This avoids that screen reader reading the word twice. 

## Screenshots

![Required](https://github.com/proudcity/wp-proudcity/assets/1569463/44134fa4-aaa4-4a1e-8561-0e6296b76acc)

**Comment by curtismchale** [comment]
@lukefretwell @kevindherman There are 3 possible settings for the form required labels. The first one `text` is what Pacific ADA is using and is the default setting. If we look at [this demo form](https://beta.proudcity.com/?gf_page=preview&id=19) it's using the `asterisk` setting. There is also a `custom` setting that allows the user to put any text at all in for the required field.

This change will test the main form settings and *only if* it's set to `text` we'll remove the `aria-required` field from the input. If it's set to `asterisk` we still need to label it required for screen readers and with `custom` I think there are too many words someone could use to indicate a required state so we should just leave it and show the `aria-required` label so that we know for sure screen readers have an indication that the field is required.

I'm going to push it to staging for test in a bit.

**Comment by curtismchale** [comment]
If you want to test:

- [this form](https://beta.proudcity.com/?gf_page=preview&id=13) should have **no** `aria-required` text
- [this form](https://beta.proudcity.com/?gf_page=preview&id=19) should have `aria-required` text

This is deploying to all `test` sites and I'm prepping to deploy to all `production` sites later today.

**Comment by curtismchale** [comment]
Deployed

Composer: https://github.com/proudcity/wp-proudcity/releases/tag/2024.09.04.0849
Builds: https://github.com/proudcity/proud-recipes/releases/tag/2024.09.04.0849
